snapshots/gapminder/2023-09-18/under_five_mortality.xlsx.dvc

```diff
@@ -0,0 +1,30 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Child Mortality Rate (Gapminder)
+    description: |-
+      Gapminder’s historic estimates compiled by Klara Johansson and Mattias Lindgren.
+    date_published: 2015-01-01
+    version_producer: v7
+    title_snapshot: Child Mortality Rate (Gapminder)
+
+    # Citation
+    producer: Gapminder
+    citation_full: Child Mortality Rate, v7, Gapminder (2015)
+    attribution_short: Gapminder (2015)
+
+    # Files
+    url_main: https://www.gapminder.org/data/documentation/gd005/
+    url_download: https://www.gapminder.org/documentation/documentation/gapdata005%20v7.xlsx
+    date_accessed: 2023-09-18
+
+    # License
+    license:
+      name: CC BY 4.0# License (same as origin.license, for backwards compatibility)
```

Comment:
License name should be "CC BY 4.0". I think there may be a bug in wizard, that adds the comment at the end.

Reply:
Yes, there was an issue with `wizard` templates. This was fixed and should work again. 

For this file I recommend having

```yaml
...
    # License
    license:
      name: CC BY 4.0
  # License (same as origin.license, for backwards compatibility)
  license:
    name: CC BY 4.0
...
```

(basically, (i) remove the comment from line 23 and copy-paste the `license` block in line 24 with one indent less)

Reply:
Why do we need `meta.license` for backwards compatibility? If there's some technical reason, then we should fix it.

Reply:
Last time I tried to run a snapshot step without the additional meta.license, it failed. Yes, if that's still the case, we should definitely fix that.

Reply:
@pabloarosado I removed the license, and it didn't fail, though it raised a warning about the missing license (that I'm removing in #1638 ). If run into it again, could you tell me how to replicate it?